src/client/common/installer/productInstaller.ts

```diff
@@ -150,6 +150,18 @@ export class DataScienceInstaller extends BaseInstaller {
                   localize.Common.bannerLabelNo()
               );
 
+        const action =
```

Comment:
Looks like the telemetry change was just made directly in release? Didn't actually realize that.